src/shared/signature.cmake

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file was removed during the Apple universal build PR as it does not correctly handle multi-arch and cross compiled builds.